pkg/helpers/mapping.go

```diff
@@ -108,3 +108,24 @@ func OneToManyFunc[K comparable, V any, mapKey comparable, Output any](keys []K,
 
 	return output
 }
+
+// // func getResSimplified
+// func getResultSimplified[K comparable, V any, mapKey comparable](key K, resultMap map[mapKey][]V) ([]V, bool) {
+// 	res, ok := resultMap[key]
+// 	return res, ok
+
+// }
+
+// getResultSimplified is a standard function to get the result of a one to many map assuming a simple map operation where the map key is the same as the dataloader key
+
+func getResultSimplified[K comparable, V any](key K, resultMap map[K][]V) ([]V, bool) {
+	res, ok := resultMap[key]
+	return res, ok
+}
+
+// OneToManyFuncSimplified uses the standard OneToManyFunc, but assumes that the mapKey is the same as the dataloaderkey K.
+func OneToManyFuncSimplified[K comparable, V any, Output any](keys []K, vals []V, getKey func(V) K, transformOutput func([]V, bool) Output) []Output {
```

Comment:
Would it make sense to have this be the "default" version of the function, and instead clarify the "non-simple" version with extra behavior (so we don't have a bunch of calls to `xyzSimple()`, and only a few calls to `xyz()`?

Reply:
That's a good call! I'll refactor it

Reply:
Thanks @ClayBenson94 , I updated the names. I also removed `func` from the other functions in the spirit of simplifying the names of these functions.